### ISSUE TITLE
Cannot login

### ISSUE THREAD

**Issue by jasonycw** [stevenlafl/threads-web-client#13]
I forked the repo and tried to deploy via Vercel (https://threads-web-client.vercel.app/)

But seems like it cannot login 
<img width="821" alt="image" src="https://github.com/stevenlafl/threads-web-client/assets/4518597/cf7ab3dc-38b6-4ce4-8d29-17e7c104e213">
 
Probably related the the npm package that this is using https://github.com/threadsjs/threads.js/issues/71

**Comment by stevenlafl** [comment]
Yep, was reported here earlier. I am not sure what the problem is exactly as it doesn't happen for me. While you're here, I recommend you pull the latest version. This fixes some duplicate loading issues, adds an emoji keyboard and will tell you when there are update on the sidebar.

**Comment by jasonycw** [comment]
I just saw your comment on https://github.com/threadsjs/threads.js/issues/71 as well, thanks for reporting that, I sync my fork and the latest code seems to still have the same issue unfortunately.

**Comment by elijah-wright** [comment]
this is an issue I experienced myself on another project lol! the issue is that Vercel's servers are located in another state (for me, Virginia), so I had to go into the app and approve the request to log in. then I could use it

**Comment by jasonycw** [comment]
Is there a way to get the token from client browser side instead of through server side API?

**Comment by stevenlafl** [comment]
That is on the radar, as it shouldn't theoretically be necessary to use the server (we'll see if there's CORS BS) - but I have it this way so I can test the frontend without making actual API calls, which would get me banned very quickly while testing. What I need to do is write a function that switches between the two based on the node environment, prod or dev.

**Comment by jasonycw** [comment]
Cool, seems like every time I try logging in with my vercel app, it triggers security alert on Instagram telling me someone outside my region accessing the account.

It would be much better if everything happens on client side without the need of server API. Or ideally, if there is web Instagram login session, just turn that to Threads session on client side.
